defaults/cslc_s1.yaml

```diff
@@ -0,0 +1,34 @@
+runconfig:
+   name: cslc_s1_workflow_default
+
+   groups:
+       pge_name_group:
+           pge_name: CSLC_S1_PGE
+
+       input_file_group:
+           # Required. List of reference and secondary S1 ZIP files
+           input_file_path:
+           # List of reference and secondary orbit file
+           orbit_file_path:
+           # List of unique burst IDs. Required if input_file_path is not provided
```

Comment:
why burst IDs required only when we don't have the input files? Can we do anything if there is no input file anyways?